api/DockerfileDev

```diff
@@ -1,10 +1,10 @@
 # local dev and dev/int/prod images are separate because they
 # are built using a different and incompatible mode (default vs release)
 
-FROM rust:1.70
+FROM rust:1.73
 
 RUN rustup target add x86_64-unknown-linux-musl
-RUN apt update && apt install -y musl-tools musl-dev postgresql-client-13
+RUN apt update && apt install -y musl-tools musl-dev postgresql-client
```

Comment:
We should keep the exact version. The rule is that it should be >= the version of the DB we have in production.